### PR TITLE
Added auth based dashboard navigation

### PR DESCRIPTION
**Pull request by aka-jain**
@deshraj @spyshiv  Please review

### CODE REVIEW

File: frontend/src/views/web/partials/dyn-header.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
       <ul class="right hide-on-med-and-down">
         <li><a class="waves-effect waves-dark" ui-sref="home" ui-sref-active="active">Home</a></li>
-        <li><a class="waves-effect waves-dark" ui-sref="web.dashboard" ui-sref-active="active">Dashboard</a></li>
+        <li ng-show="isAuth"><a class="waves-effect waves-dark" ui-sref="web.dashboard" ui-sref-active="active">Dashboard</a></li>
```
Comment (spyshiv) [inline frontend/src/views/web/partials/dyn-header.html]: @aka-jain : I think this list item should be after the comment (#L13). That make sense with code readability.

Comment (aka-jain) [inline frontend/src/views/web/partials/dyn-header.html]: Sure.

**Comment by aka-jain** [comment]
@spyshiv : Changes done, anything else?

**Comment by spyshiv** [comment]
LGTM . :+1:

**Comment by deshraj** [comment]
@spyshiv are you merging this?

**Comment by spyshiv** [comment]
By mistake i hit the `close and button`. Merging this

**Comment by deshraj** [comment]
Sounds good. :-)

**Pull request merged**